pkgs/os-specific/linux/kernel/linux-4.11-kfd.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, hostPlatform, fetchFromGitHub, perl, buildLinux, ... } @ args:
+
+let
+  ver = "4.11.0";
+  revision = "kfd-roc-1.6.x";
+in
+import ./generic.nix (args // rec {
+  version = "${ver}-${revision}";
+  modDirVersion = "${ver}";
+  extraMeta.branch = "4.11";
+
+  src = fetchFromGitHub {
+    owner = "RadeonOpenCompute";
+    repo = "ROCK-Kernel-Driver";
+    # url = "https://github.com/RadeonOpenCompute/ROCK-Kernel-Driver.git";
+    rev = "roc-1.6.x";
```

Comment:
This is a branch, can you use a `tag` instead? Otherwise the source hash will break.

Reply:
The latest tag is for 1.5 it seems, perhaps that is too old? In that case, using the commit id might be a little better than just the branch, until a tag is created.